### ISSUE TITLE
config exported using ros config export is invalid

### ISSUE THREAD

**Issue by leodotcloud** [rancher/os#2337]
```
[rancher@rancher ~]$ sudo ros config export
EXTRA_CMDLINE: /init
rancher:
  environment:
    EXTRA_CMDLINE: /init
  network:
    interfaces:
      eth0:
        address: 1.1.1.2/29
        dhcp: false
        gateway: 1.1.1.1
  password: blahblahblah
  services_include:
    open-vm-tools: true
  state:
    dev: LABEL=RANCHER_STATE
    wait: true
ssh_authorized_keys: []
```

```
[rancher@rancher ~]$ sudo ros config export > /tmp/c.yaml
[rancher@rancher ~]$ sudo ros config validate -i /tmp/c.yaml
ERRO[0000] EXTRA_CMDLINE: Additional property EXTRA_CMDLINE is not allowed
ERRO[0000] password: Additional property password is not allowed
[rancher@rancher ~]$
```

```
[rancher@rancher ~]$ system-docker ps -a
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/system-docker.sock: Get http://%2Fvar%2Frun%2Fsystem-docker.sock/v1.31/containers/json?all=1: dial unix /var/run/system-docker.sock: connect: permission denied
[rancher@rancher ~]$ sudo system-docker ps -a
CONTAINER ID        IMAGE                              COMMAND                  CREATED             STATUS                     PORTS               NAMES
db6668ac0262        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Exited (0) 8 minutes ago                       preload-user-images
b8ac2f88a31c        rancher/os-docker:17.09.1          "ros user-docker"        8 minutes ago       Up 8 minutes                                   docker
902bef4bc94a        rancher/os-console:v1.3.0          "/usr/bin/ros entr..."   8 minutes ago       Up 8 minutes                                   console
4a75892d0c38        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Exited (0) 8 minutes ago                       cloud-init-execute
fd3cc215bd00        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Up 8 minutes                                   ntp
ec639f3b81d9        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Up 8 minutes                                   network
434b01a780b6        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Up 8 minutes                                   udev
3be4a5661abb        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Created                                        all-volumes
ec019f76cd4e        rancher/container-crontab:v0.4.0   "container-crontab"      8 minutes ago       Up 8 minutes                                   system-cron
cf22e12e1068        rancher/os-acpid:v1.3.0            "/usr/bin/ros entr..."   8 minutes ago       Up 8 minutes                                   acpid
37d567cac9ae        rancher/os-logrotate:v1.3.0        "/usr/bin/entrypoi..."   8 minutes ago       Exited (0) 7 minutes ago                       logrotate
fba0acad5b07        rancher/os-syslog:v1.3.0           "/usr/bin/entrypoi..."   8 minutes ago       Up 8 minutes                                   syslog
f18be4640125        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Exited (0) 8 minutes ago                       udev-cold
9a543671cb65        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Created                                        system-volumes
f89f6885e303        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Created                                        user-volumes
f6c01c18d1ba        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Created                                        media-volumes
ec86078fb1bb        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Created                                        command-volumes
726ce90a348b        rancher/os-base:v1.3.0             "/usr/bin/ros entr..."   8 minutes ago       Created                                        container-data-volumes
[rancher@rancher ~]$
```

**Comment by niusmallnan** [comment]
`rancher.password` should be removed in `ros config export`, because it is a kernel parameter.

**Comment by kingsd041** [comment]
Fixed in rancheros -v1.4.0-rc1
